internal/inspect/utils.go

```diff
@@ -0,0 +1,68 @@
+package inspect
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/go-errors/errors"
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgx/v4"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/db/reset"
+	"github.com/supabase/cli/internal/utils"
+)
+
+const ROOT_DIR = "queries/"
+const CSV_QUERY = `COPY (%s) TO STDOUT WITH CSV HEADER`
+
+//go:embed queries/*.sql
+var queries embed.FS
```

Comment:
The problem with embed.FS is that misspelt paths become runtime errors. This would have been fine if we had unit test coverage for all inspect commands. Since we are doing reliability month, could you look into adding them?

Another alternative is to embed each file as a separate variable.

Reply:
Will look into adding unit tests, yep!